var/spack/repos/builtin/packages/py-bokeh/package.py

```diff
@@ -12,6 +12,7 @@ class PyBokeh(PythonPackage):
     homepage = "https://github.com/bokeh/bokeh"
```

Comment:
```suggestion
    homepage = "https://bokeh.org/"
```
We try to be consistent with what is specified on PyPI.